lib/util/src/platform.c

```diff
@@ -208,7 +208,7 @@ hse_platform_init(void)
     merr_t err;
 
     if (PAGE_SIZE != getpagesize()) {
-        log_err("compile-time PAGE_SIZE (%lu) != run-time getpagesize (%d)",
+        log_err("compile-time PAGE_SIZE (%u) != run-time getpagesize (%d)",
```

Comment:
I think the type of PAGE_SIZE is unsigned long from the headers, right?  You're asking for trouble if you're changing it to uint (i.e., calculations that assume it's unsigned long might now not produce the correct result).

Reply:
this actually fixed a bug somewhere. Yay

Reply:
Nasty..